leafsmart-front-end/package.json

```diff
@@ -36,5 +36,6 @@
       "last 1 firefox version",
       "last 1 safari version"
     ]
-  }
+  },
+  "proxy": "http://localhost:3000"
```

Comment:
Proxy is for letting the server know where to look for any HTTP requests that don't have a full address, like `/login`, right? Should this be localhost:3001 where our server's living in development?